extensions/git/package.nls.json

```diff
@@ -1,6 +1,8 @@
 {
 	"displayName": "Git",
 	"description": "Git SCM Integration",
+	"command.continueInLocalClone": "Clone Repository Locally and Open on Desktop...",
```

Comment:
Is "Open on Desktop" the terminology that we are using in the "ConntinueOn" flows?

Reply:
`Clone Repository Locally and Open on Desktop...` is the exact name that we use for the Remote Repositories' contribution (which does the same thing but for virtual workspaces). I'm open to alternative suggestions for the name though, as I have been tweaking several other option names based on team feedback.